src/vs/editor/contrib/suggest/media/suggest.css

```diff
@@ -239,6 +243,14 @@
 	white-space: initial;
 }
 
+.monaco-editor .suggest-widget .details > .monaco-scrollable-element > .body > .docs.markdown-docs > div > p:first-child {
+	margin-top: 0;
+}
+
+.monaco-editor .suggest-widget .details > .monaco-scrollable-element > .body > .docs.markdown-docs > div > p:last-child	 {
```

Comment:
Just curious, why do you need to use a specific rule like `.monaco-editor .suggest-widget .details > .monaco-scrollable-element > .body > .docs.markdown-docs > div > p:last-child`? isn't it sufficient enough to have Just curious, why do you need to use a specific rule like `.monaco-editor .suggest-widget .details > .monaco-scrollable-element > .body > .docs.markdown-docs p:last-child`?

Reply:
@soleo I think there was some copy/paste problem with your comment above